src/ln/channelmonitor.rs

```diff
@@ -675,6 +699,20 @@ impl ChannelMonitor {
 			writer.write_all(&[0; 1])?;
 		}
 
+		if let Some(ref prev_latest_per_commitment_point) = self.prev_latest_per_commitment_point {
+			writer.write_all(&prev_latest_per_commitment_point[..])?;
+		} else {
+			writer.write_all(&[0;32])?;
+
+		}
+
+		if let Some(ref latest_per_commitment_point) = self.latest_per_commitment_point {
+			writer.write_all(&latest_per_commitment_point[..])?;
+		} else {
+			writer.write_all(&[0;32])?;
+
+		}
+
```

Comment:
you don't want to persist secret?  also, seems it's better to re-sign local_commitment_tx can when it's loaded onto the memory and strip off signature when it's stored into hard disk.

Reply:
If I get derivation scheme right you can't do nothing with a per_commitment_point, you still need basepoint and basepoint_secret? ChannelMonitor handling current per_commitment_point is to avoid KeysManager or whatsever having to remember the per_commitment_point for each new commitment, without knowing if it's gonna be useful.
Hard disk storage thing is under revision, so here it's temporary.